shared/ui-components/src/page_components/UnitTables.tsx

```diff
@@ -3,7 +3,7 @@ import { nanoid } from "nanoid"
 import { MinMax, UnitSummary, Unit } from "@bloom-housing/core"
 
 import { BasicTable } from "../tables/BasicTable"
-import t from "../../src/helpers/translator"
+import { t } from "../../src/helpers/translator"
```

Comment:
Pedantic, but this should be:

```suggestion
import { t } from "../helpers/translator"
```

Reply:
and the next two lines, too